diff -urN a/plugins/actor/JESS/draw.c b/plugins/actor/JESS/draw.c
--- a/plugins/actor/JESS/draw.c	2023-03-20 03:04:42.000000000 +0100
+++ b/plugins/actor/JESS/draw.c	2026-08-02 21:49:09.430350297 +0200
@@ -69,7 +69,7 @@
 		while (priv->life[i] > 0)
 		{
 			i++;
-			if (i == FUSEE_MAX)
+			if (i == FUSEE_MAX) /* not FUSEE_MAX+1: last valid slot is FUSEE_MAX-1 */
 				return;
 		}
 		priv->xi[i] = visual_random_context_int(priv->rcontext) % priv->resx - priv->xres2;
